documentation/tutorial.md

```diff
@@ -68,3 +71,416 @@ tutorial overlord    localhost/localhost  2.9.37   unsupported  23:20:53Z
+tutorial  overlord    localhost/localhost  2.9.37   unsupported  14:44:25Z
+
+App      Version  Status  Scale  Charm    Channel   Rev  Exposed  Message
+mongodb           active      2  mongodb  dpe/edge   96  no       Replica set primary
+
+Unit        Workload  Agent  Machine  Public address  Ports      Message
+mongodb/0*  active    idle   0        10.23.62.156    27017/tcp  Replica set primary
+mongodb/1   active    idle   1        10.23.62.55     27017/tcp  Replica set secondary
+
+Machine  State    Address       Inst id        Series  AZ  Message
+0        started  10.23.62.156  juju-d35d30-0  focal       Running
+1        started  10.23.62.55   juju-d35d30-1  focal       Running
+
+```
+
+As previously mentioned you can trust that Charmed MongoDB removed this replica correctly.
+
+
+## Passwords:
+### Retrieving the admin password:
+As previously mentioned, the admin password can be retrieved by running the `get-admin-password` action on the Charmed MongoDB application:
+```
+juju run-action mongodb/leader get-admin-password --wait
+```
+Running the command should output:
+```
+unit-mongodb-0:
+  UnitId: mongodb/0
+  id: "2"
+  results:
+    admin-password: <password>
+  status: completed
+  timing:
+    completed: 2022-12-02 11:30:01 +0000 UTC
+    enqueued: 2022-12-02 11:29:57 +0000 UTC
+    started: 2022-12-02 11:30:01 +0000 UTC
+```
+The admin password is under the result: `admin-password`.
+
+
+### Rotating the admin password
+You can change the admin password to a new random password by entering:
+```
+juju run-action mongodb/leader set-admin-password --wait
+```
+Running the command should output:
+```
+unit-mongodb-0:
+  UnitId: mongodb/0
+  id: "4"
+  results:
+    admin-password: <new password>
+  status: completed
+  timing:
+    completed: 2022-12-02 14:53:30 +0000 UTC
+    enqueued: 2022-12-02 14:53:25 +0000 UTC
+    started: 2022-12-02 14:53:28 +0000 UTC
+```
+The admin password is under the result: `admin-password`. It should be different from your previous password.
+
+*Note when you change the admin password you will also need to update the admin password the in MongoDB URI; as the old password will no longer be valid*
+
+### Setting the admin password
+You can change the admin password to a specific password by entering:
+```
+juju run-action mongodb/leader set-admin-password password=<password> --wait
+```
+Running the command should output:
+```
+unit-mongodb-0:
+  UnitId: mongodb/0
+  id: "4"
+  results:
+    admin-password: <password>
+  status: completed
+  timing:
+    completed: 2022-12-02 14:53:30 +0000 UTC
+    enqueued: 2022-12-02 14:53:25 +0000 UTC
+    started: 2022-12-02 14:53:28 +0000 UTC
+```
+The admin password under the result: `admin-password` should match whatever you passed in when you entered the command.
+
+*Note when you change the admin password you will also need to update the admin password the in MongoDB URI; as the old password will no longer be valid*
+
+
```

Comment:
Maybe I would also show off the juju commands to list all actions that have now been executed on our deployment. Was it somehting like "juju operations" ? (I think we actions v2 interface, but I suppose there is the same in the v1 interface, isn't it?)

Reply:
Not sure:
```
juju operations
ERROR juju: "operations" is not a juju command. See "juju --help".
```

But also I don't think this is necessary for learning how to use Charmed MongoDB. We could also just do `history | grep juju`

Reply:
I believe you should first change the API interface using 

```
export JUJU_FEATURES=actions-v2
```

I believe I saw this in Hello Kubecon workshop by Jon 

Anyhow, there should be a way to see this in the actions-v1 interface too I suppose. Maybe juju list-*?

Reply:
Hmm; regardless of this I don't think seeing the previously used Juju commands adds much to the tutorial